src/bridge/lib/magic_numbers.py

```diff
@@ -18,6 +18,7 @@
 DAYS_IN_MONTH = 30
 DEFAULT_DNS_PORT = 53
 DEFAULT_ELASTICSEARCH_PORT = 9200
+DEFAULT_FASTLY_BACKEND_TTL = 86400
```

Comment:
I'd say it's actually more useful to indicate what unit of time that number represents, because it's still fairly enigmatic. Also, by tying it to Fastly it reduces its reusability.